### PR TITLE
4.3.0rc2

### PR DESCRIPTION
**Pull request by phoebusm**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by phoebusm** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "4.3.0rc2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 8f9d65c37bc586443f88aa525c0f0a66da65b0b38a3f04a38bebd32e85dd22a2
+  sha256: 32e8cd339e0c8e22f9fbe38a5afd4208e6eb2cec4994d6561e47be6998d1fbc2
```
Comment (jjerphan) [inline recipe/meta.yaml:10]:
```suggestion
  sha256: 21a96ba6ef5d690b23b67a1b7903e311319194aaea01b6d307ef4f551e5f9621
```

Comment (phoebusm) [reply recipe/meta.yaml]: Weird. How did I get the wrong checksum?
Thanks for the update.

**Review by poodlewars — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (jjerphan) [reply recipe/meta.yaml]: I think the one reported on GitHub is different than the one of the tarball. I tend to verify it with `sha256sum`.

**Pull request merged**